.github/workflows/android-build.yml

```
@@ -72,6 +72,9 @@ jobs:
           # Create dummy libfreetype.a file to satisfy the build dependency
           touch external/freetype2/prebuilt/android/armeabi/libfreetype.a
           
+          # Create dummy libcurl.a file to satisfy the build dependency
+          touch external/curl/prebuilt/android/armeabi/libcurl.a
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)